.changeset/big-games-deny.md

```diff
@@ -0,0 +1,12 @@
+---
+"@logto/integration-tests": minor
+"@logto/experience": minor
+"@logto/console": minor
+"@logto/shared": minor
+"@logto/core": minor
+---
+
+add phone number validation/parser to ensure valid format when updating existing users' primary phone or creating new users with a phone number
```

Comment:
add phone number validation and parsing to ensure the correct format when updating an existing user’s primary phone number or creating a new user with a phone number